drivetrain.py

```diff
@@ -65,11 +68,11 @@ def mix_channels(self, pulse_throttle, pulse_steering):
         self.set_servo_pulse(self.channels['left'], output_pulse_left)
         self.set_servo_pulse(self.channels['right'], output_pulse_right)
 
-    def map_channel_value(self, value, min_max_range):
+    def _map_channel_value(self, value):
```

Comment:
Fairly self explanitory, but just needs a one liner comment staying what this method does.